src/improvements/tasks/eth/002-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,268 @@
+```
+
+### Decode Multicall3DelegateCall calldata:
+```bash
+cast calldata-decode 'aggregate3((address,bool,bytes)[])' <0x82ad56cb...>
+
+[
+    (
+        0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76,
+        false, 
+        0xff2dd5a100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000c407398d063f942febbcc6f80a156b47f3f1bda60000000000000000000000003b73fa8d82f511a3cae17b5a26e4e1a2d5e2f2a4039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9
+    )
+]
+```
+
+1. First tuple (Call3 struct for Multicall3DelegateCall)
+    - `target`: [0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/validation/standard/standard-versions-mainnet.toml#L21) - Mainnet OPContractsManager v2.0.0
+    - `allowFailure`: false
+    - `callData`: `0xff2dd5a1...` See below for decoding.
+        - Command to encode: `cast calldata 'aggregate3((address,bool,bytes)[])' "[(0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76,false,0xff2dd5a100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000c407398d063f942febbcc6f80a156b47f3f1bda60000000000000000000000003b73fa8d82f511a3cae17b5a26e4e1a2d5e2f2a4039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9)]"`
+
+### Decode upgrade calldata
+
+```bash
+cast calldata-decode 'upgrade((address,address,bytes32)[])' <0xff2dd5a1...>
+
+[
+    (
+        0xc407398d063f942feBbcC6F80a156b47F3f1BDA6,
+        0x3B73Fa8d82f511A3caE17B5a26E4E1a2d5E2f2A4,
+        0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9
+    )
+]
+```
+1. First tuple (Unichain):
+    - SystemConfigProxy: [0xc407398d063f942feBbcC6F80a156b47F3f1BDA6](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/superchain/configs/mainnet/unichain.toml#L58)
+    - ProxyAdmin: [0x3B73Fa8d82f511A3caE17B5a26E4E1a2d5E2f2A4](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/superchain/configs/mainnet/unichain.toml#L59)
+    - AbsolutePrestate: [0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9](https://github.com/ethereum-optimism/optimism/blob/63da401391e9be93517d242da5da24905aa5b84c/op-program/prestates/releases.json#L9)
+    - Command to encode: `cast calldata 'upgrade((address,address,bytes32)[])' "[(0xc407398d063f942feBbcC6F80a156b47F3f1BDA6,0x3B73Fa8d82f511A3caE17B5a26E4E1a2d5E2f2A4, 0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9)]"`
+
+## Tenderly State Changes
+[Link](https://dashboard.tenderly.co/oplabs/eth-mainnet/simulator/42687998-3100-49a8-bf77-7448c707f2b1)
```

Comment:
Is including a simulation link new? I think signers should just use the one printed from their terminal no?

Reply:
Oh, also I don't think this link is usable outside of the op labs account.